pagopa/io-app
# chore: [IA-874] Switch support CTA order

## Short description
This PR switches the order of the CTA in the support component.

## List of changes proposed in this pull request
- Inverted the CTA order in the `ZendeskSupportComponent`

| Before      | After |
| ----------- | ----------- |
| <img src="https://user-images.githubusercontent.com/11773070/171379271-fbe2b3c2-f3a6-4e3d-bc3a-6912bede6aa5.png" width=300>     | <img src="https://user-images.githubusercontent.com/11773070/171379265-6b815ac3-f0c5-4003-b3fa-471e457cdfef.png" width=300>       |

## How to test
Open the assistance component by pressing the "?" button.
Note that to see the "See your request" button you should do at least a request.
